pkg/cmd/dev/bench.go

```diff
@@ -30,7 +30,8 @@ func makeBenchCmd(runE func(cmd *cobra.Command, args []string) error) *cobra.Com
 		Long:  `Run the specified benchmarks.`,
 		Example: `
 	dev bench pkg/sql/parser --filter=BenchmarkParse
-	dev bench pkg/bench -f='BenchmarkTracing/1node/scan/trace=off' --count=2 --bench-time=10x --bench-mem`,
+	dev bench pkg/bench -f='BenchmarkTracing/1node/scan/trace=off' --count=2 --bench-time=10x --bench-mem
+	dev bench pkg/bench -f='BenchmarkFoo' --ignore-cache --test-args='-test.cpuprofile=cpu.out -test.memprofile=mem.out'`,
```

Comment:
If you can replace BenchmarkFoo with something real that'd be slightly better, cause you could run it and see the .out files appear.

Reply:
I didn't want the line to get so long that it'd scroll off a terminal, but I guess I'll just use the example above.